<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Журнал Административных распоряжений</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon/icon.png</normaloff>:/icon/icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelFind">
        <property name="text">
         <string>Поиск</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditFind">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Строка поиска в поле тема</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonFind">
        <property name="toolTip">
         <string>Найти распоряжения (Ctrl+F)</string>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icon/find.png</normaloff>:/images/icon/find.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+F</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonMoreOptions">
        <property name="toolTip">
         <string>Показать/скрыть дополнительные настройки фильтра поиска</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icon/less.png</normaloff>:/images/icon/less.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="toolTip">
         <string>Добавить новую запись (Insert)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icon/add.png</normaloff>:/images/icon/add.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ins</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonEdit">
        <property name="toolTip">
         <string>Редактировать/Просмотреть выделенную запись (Ctrl+E)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icon/edit.png</normaloff>:/images/icon/edit.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Ctrl+E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDelete">
        <property name="toolTip">
         <string>Удалить выделенную запись (Del)</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/icon/remove.png</normaloff>:/images/icon/remove.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="shortcut">
         <string>Del</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="widgetExtendedOptions" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelNumDoc">
           <property name="text">
            <string>Номер</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditNumDoc">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="toolTip">
            <string>Строка поиска в поле номер</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelDataFrom">
           <property name="text">
            <string>Дата с</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditFrom">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelDataTo">
           <property name="text">
            <string>по</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditTo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxDate"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxForInitiated">
         <property name="toolTip">
          <string>Отобразить распоряжения предназначенные мне для ознакомления/все</string>
         </property>
         <property name="text">
          <string>Показать распоряжения для ознакомления</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxShowDeleted">
         <property name="toolTip">
          <string>Отображать/нет удаленные распоряжения</string>
         </property>
         <property name="text">
          <string>Показывать удаленные распоряжения</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreateShortcut"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="actionJob"/>
    <addaction name="actionUsers"/>
    <addaction name="actionMainSettings"/>
    <addaction name="actionChangePassword"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionJob">
   <property name="text">
    <string>Должности</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="actionUsers">
   <property name="text">
    <string>Пользователи</string>
   </property>
  </action>
  <action name="actionMainSettings">
   <property name="text">
    <string>Общие настройки</string>
   </property>
  </action>
  <action name="actionChangePassword">
   <property name="text">
    <string>Сменить пароль</string>
   </property>
  </action>
  <action name="actionCreateShortcut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Создать ярлык на рабочем столе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
